Dockerfile

```diff
@@ -2,15 +2,30 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
-FROM golang:1.13.4-alpine AS builder
-
+FROM golang:1.13.4 AS tester
+ARG GOPROXY=https://proxy.golang.org
```

Comment:
This shouldn't be needed with Go 1.13, as there's now a default value for GOPROXY.

Reply:
Yeah! Rebased from master, that has this improvement too